### ISSUE TITLE
Streaming Audio notification keeps closing by itself when paused

### ISSUE THREAD

**Issue by Pentaphon** [meganz/android#176]
## General Information

- **MEGA Version:** 6.0

- **OS Version(s):**  Android 11

- **Devices/Simulators:** Moto G9

- **Reproducible? (Yes/No):** Unpredictable 

##  What happened? 
The audio notification keeps closing when streaming audio is paused and I use my phone for something else like taking calls, forcing me to open the app again the next time I want to stream audio, instead of staying active so I can continue streaming audio from the notification. I suspect this is because Mega keeps being kicked out of memory but the audio notification should stay in place no matter what like in other audio players such as podcast clients.

## What did you expect to happen?
I expected the audio notification to stay in place when streaming audio is paused unless I close it myself.

**Comment by javiergm1983** [comment]
Hi @Pentaphon I have tested and cannot reprouce your issue.
Do you have the setting "Media > Audio files > Playing on the background" enabled?

Please, could you attach a screen recording reproducing the issue? Thanks.

**Comment by Pentaphon** [comment]
Yes, I have that enabled but I don't think I will be able to screen record the issue because it is very unpredictavle. I am looking more for a way to make sure the player notification to stay up no matter what instead of it being possible to have it close by itself. I notice the notification goes into "silent mode" when it is paused. Not sure if that has anything to do with it. I also don't know why the notification isn't grouped with other audio notifications like they are on Android 11.

**Comment by javiergm1983** [comment]
Do you have any battery saver enabled?

**Comment by Pentaphon** [comment]
> Do you have any battery saver enabled?

No, but battery optimization was enabled for the MEGA app. I have just changed it to "not optimized" and I will continue to monitor MEGA to see if the audio notification still closes by itself when I do something else on my phone.

**Comment by Pentaphon** [comment]
> Do you have any battery saver enabled?

After having the Mega app "not optimized" for a few days I can definitely say that the audio notification does not stay in place while doing other things on my phone. Not sure why it happens but it would be nice to have a way to keep the notification alive when it is paused.

**Comment by javiergm1983** [comment]
Seems that our QA team cannot also reproduce the issue... Here is what they reported for now.

> I tried to reproduce the issue in a Samsung Android 11 device. i can´t reproduce the issue. You can resume the audio using the notification button.
The only difference between MEGA and streaming audio providers (like , spotify or pocket cast). It is that when you turn off the notification that interrrupted the audio, in MEGA it does not play automatically, but I can resume the audio using the "play" button of the notification

**Comment by Pentaphon** [comment]
> I tried to reproduce the issue in a Samsung Android 11 device.

That device probably has a lot of RAM. My device has 3GB of RAM so maybe on lower-RAM devices, Mega doesn't survive for long.

I didn't expect a successful reproduction of my issue since there's no specific way to make the Mega notification go away. It just goes away by itself with normal use for me when it is paused. 

Perhaps there is something else that can be done to ensure the Mega notification stays in place no matter what? Something to keep it from getting kicked out of memory? 

Also, I must ask, why isn't the Mega notification grouped with the other audio notifications for other apps on Android 11? This was a big change in Android 11 that groups all my audio notifications together but for some reason, the Mega audio notification isn't included? Does this mean Mega isn't totally Android 11 ready? Here is how audio notifications are supposed to be grouped in Android 11 devices according to the Google Developers blog. 

https://android-developers.googleblog.com/2020/08/playing-nicely-with-media-controls.html

![media-controls](https://user-images.githubusercontent.com/49374058/151104527-61c552b1-37bf-4e95-bf16-39f55bcbdafc.gif)
